app/controllers/log_entries_controller.rb

```diff
@@ -24,7 +25,7 @@ def destroy
 
   def index
     @entry = LogEntry.new
-    @entries = LogEntry.includes(:user).order('created_at desc')
+    @entries = LogEntry.includes(:user).where(pinned: true).order('created_at desc')
```

Comment:
```suggestion
    @entries = LogEntry.includes(:user).order(pinned: :desc, created_at: :desc)
```
We just have to add a primary order clause to get pinned ones to show up first